### PR TITLE
Hdk v0.0.138

### PR DESCRIPTION
**Pull request by JettTech**

### CODE REVIEW

File: version-manager.json [hunk 1]
```diff
@@ -1,8 +1,7 @@
...
-    "hc_utils": "0.0.126",
-    "holo_hash": "0.0.21",
-    "holochain": "0.0.131",
-    "holochain_rev": "v0_0_131",
-    "holonix_rev": "44542b5c0cdc984454fad96fbec313b7b1c8ca7c"
+    "hdk": "0.0.138",
+    "hc_utils": "0.0.138",
+    "holo_hash": "0.0.29",
+    "holochain_rev": "holochain-0.0.145",
+	"lair_rev": "lair_keystore-v0.2.0"
```
Comment (zo-el) [inline version-manager.json:6]:
```suggestion
	"lair_rev": "v0.0.10"
```

File: zomes/test-integrity/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+[package]
+name = "test_integrity"
+version = "0.4.4"
```
Comment (zo-el) [inline zomes/test-integrity/Cargo.toml:3]:
```suggestion
version = "0.1.0"
```

File: zomes/test-integrity/src/validation.rs [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    }
+    false
+}
+
+#[hdk_extern]
+fn genesis_self_check(data: GenesisSelfCheckData) -> ExternResult<ValidateCallbackResult> {
+    if is_skipped() {
+        Ok(ValidateCallbackResult::Valid)
+    } else {
+        // Ok(ValidateCallbackResult::Valid)
```
Comment (zo-el) [inline zomes/test-integrity/src/validation.rs:31]:
```suggestion
```

File: zomes/test/Cargo.toml [hunk 4]
```diff
@@ -2,14 +2,15 @@
...
+edition = "2021"
 
 [lib]
 name = "test"
 crate-type = [ "cdylib", "rlib" ]
 
 [dependencies]
-hdk = "=0.0.126"
-serde = "1.0.104"
+hdk = {version = "=0.0.138"}
```
Comment (zo-el) [inline zomes/test/Cargo.toml:12]:
```suggestion
hdk = "=0.0.138"
```

**Review by zo-el — APPROVED** [review]
Small changes requested. Else everything looks good

Also another big request is to upgrade the update script in the MakeFile.
Please check out the holofuel repo for what was required: https://github.com/Holo-Host/holofuel/blob/3bbea385523b42549005e0f13f474928358b18b4/Makefile#L104-L109

Just need to upgrade the way nix is updated in the repo

File: tests/package.json [hunk 5]
```diff
@@ -1,13 +1,21 @@
...
-  "type": "module",
+  "authors": [
+    "zo-el<joelulahanna@gmail.com>",
+    "jetttech<lisa.jetton@holo.host>"
+  ],
+  "license": "ISC",
   "dependencies": {
     "@holo-host/cryptolib": "^0.3.2",
-    "@holochain/tryorama": "^0.5.8",
-    "tap-diff": "^0.1.1",
```
[the remark this answers is not in the document]
Comment (JettTech) [reply tests/package.json:10]: I don't see that it was actually ever used in this repo, but I'm happy to add it to the script, as it *is* super helpful with debugging. :+1:

**Pull request merged**